packages/shared/src/components/cards/common/CardOverlay.tsx

```diff
@@ -18,7 +20,14 @@ const CardOverlay = ({
     return null;
   }
 
-  return <CardButton title={post.title} onClick={onPostCardClick} />;
+  return (
```

Comment:
Oh interesting, I kind of thought it was a a already, looking at list card for instance 😅
Maybe we can unify the two into 1 component now?